packet_handler_map_test.go

```diff
@@ -63,6 +66,7 @@ var _ = Describe("Packet Handler Map", func() {
 	})
 
 	JustBeforeEach(func() {
+		log.SetOutput(ioutil.Discard)
```

Comment:
This is not thread-safe, so I would honestly do it at the start of TestMain and not worry about setting it back to anything else. I assume none of the tests should log directly to stderr, anyway.